geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessengerJUnitTest.java

```diff
@@ -484,6 +498,38 @@ public void testBigMessageIsFragmentedWhenBroadcast() throws Exception {
     doTestBigMessageIsFragmented(false, true);
   }
 
+  public static class ByteHolder implements DataSerializableFixedID {
+    static final int DSFID = 123456;
+    private byte[] payload;
+
+    @Override
+    public int getDSFID() {
+      return DSFID;
+    }
+
+    public ByteHolder(byte[] payload) {
+      this.payload = payload;
+    }
+
+    public ByteHolder() {}
+
+    @Override
+    public void toData(DataOutput out, SerializationContext context) throws IOException {
+      StaticSerialization.writeByteArray(payload, out);
+    }
+
+    @Override
+    public void fromData(DataInput in, DeserializationContext context)
```

Comment:
ClassNotFoundException isn't ever thrown (IntelliJ warning).

Reply:
We probably have a lot of classes that IJ would give this warning about.